### PR TITLE
Implement small test

### PR DESCRIPTION
**Pull request by FaBioLeeNA**

### CODE REVIEW

File: server/fakeData/fakeDataHelpers.js [hunk 1]
```diff
@@ -40,11 +40,11 @@ const createFakeCategories = (n) => {
   return categories;
 }
 
-const fillCategories = (n) => {
+const fillCategories = async (n) => {
```
Comment (NMendini) [inline server/fakeData/fakeDataHelpers.js:43]: I like your use of async/await here. It is a good choice to help prevent an overload of database queries. Great job!

File: server/fakeData/fakeDataHelpers.js [hunk 2]
```diff
@@ -24,11 +24,11 @@ const fillProductTypes = async (n) => {
   for (var i = 0; i < productTypes.length; i++) {
```
Comment (NMendini) [inline server/fakeData/fakeDataHelpers.js:24]: My only criticism at this point would be the commit message for not being very specific, but I must say I found it quite amusing 😄

**Review by NMendini — APPROVED** [review]
I think your code looks clean and from the commits it looks like you've already done a little maintenance and improvement. Way to go!

File: server/fakeData/fakeDataHelpers.js (hunk 1 above)
Comment (FaBioLeeNA) [reply server/fakeData/fakeDataHelpers.js:43]: Thank you for the feedback!

File: server/fakeData/fakeDataHelpers.js (hunk 2 above)
Comment (FaBioLeeNA) [reply server/fakeData/fakeDataHelpers.js:24]: Thank you. I will try to keep improving my commits messages.

**Comment by FaBioLeeNA** [comment]
https://trello.com/c/Sk0muHqG

**Pull request merged**